pbl

```diff
@@ -283,7 +311,8 @@ if(defined $lang) {
 if($program ne 'pbl') {
   # compat: update-bootloader or bootloader_entry
   push @todo, [ 'config' ];
-  unshift @todo, [ 'install' ] if grep { /^--reinit$/ } @ARGV;
+  #  moved to GetOptions handling
+  #  unshift @todo, [ 'install' ] if grep { /^--reinit$/ } @ARGV;
```

Comment:
well, I would remove it completely. I think dead code is not needed.

Reply:
It is a dead comment :-)